src/to-have-style.js

```diff
@@ -21,7 +21,8 @@ function isSubset(styles, computedStyle) {
     !!Object.keys(styles).length &&
     Object.entries(styles).every(
       ([prop, value]) =>
-        computedStyle.getPropertyValue(prop.toLowerCase()) === value,
+        computedStyle[prop] === value ||
+        computedStyle[prop.toLowerCase()] === value,
```

Comment:
`computedStyle.getPropertyValue` supports only dash-cased notation. Property getters work for both dash and camelCase